iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -78,6 +78,7 @@ private InputFormatConfig() {
   public static final String CTAS_TABLE_NAME = "iceberg.mr.ctas.table.name";
   public static final String SELECTED_COLUMNS = "iceberg.mr.selected.columns";
   public static final String EXTERNAL_TABLE_PURGE = "external.table.purge";
+  public static final String METADATA_LOCATION = "metadata_location";
```

Comment:
This feels problematic to me, as it is already stored in the HiveTableOperations